terraform/spec/fixtures/projects/provider_with_multiple_local_path_modules/loader/project/providers.tf

```diff
@@ -0,0 +1 @@
+../providers.tf
```

Comment:
This seems to be some kind of terraform magic reference, vscode also renders it as the referenced file, not sure how this works yet

Reply:
Looks like this when viewed in vscode 🤔

<img width="770" alt="Screenshot 2021-07-14 at 15 12 46" src="https://user-images.githubusercontent.com/20165/125637008-9638859b-952d-427a-aa4a-ee19bb3a49f5.png">

Reply:
![magic](https://user-images.githubusercontent.com/80475/125648180-538928c7-57fa-4e99-9d7e-d0fef81442bd.gif)